css/style.css

```diff
@@ -432,18 +440,33 @@ th {
 }
 
 .container {
-  width: 60%;
+  width: 65%;
   display: flex;
   justify-content: space-between;
 }
 
+@media (max-width: 56.25em) {
+  .container {
```

Comment:
I find that using `em` for breakpoints makes it harder to understand. Consider using pixels which are more predictable and are widely used in media queries.

Reply:
I read something about using px and rem in media query in the link: https://zellwk.com/blog/media-query-units/
If you read the following on the page: (2 and 3)
2. User Zooms In
The original idea behind em based sizes was due to older browsers not being able to update pixel values when a user zooms. In this regard, testing the difference between media query units when a user zooms will help to answer the question on whether we can use px based media queries now.
3. User Changed Their Browser’s Font Setting

Concluding The Experiments
As you can see from our tests above, the only unit that performed consistently across all four browsers is em. There aren’t any differences between em and rem with the exception of bugs found on Safari.

px media queries performed well in two of the three experiments (with the exception of Safari, again). Unfortunately, px media queries remained at 400px in the third experiment, which makes it a no-go if you intend to support users who change their browser’s font-size value.

Hence, my conclusion after these experiments is: Use em media queries.